src/main/java/com/teamscale/upload/xcode/XCResultConverter.java

```diff
@@ -307,5 +406,17 @@ public ConversionException(String message) {
 		public ConversionException(String message, Exception e) {
 			super(message, e);
 		}
+
+		/**
+		 * Creates a {@link ConversionException} with the given message and the
+		 * {@linkplain ProcessResult#errorOutput error output of the command}.
+		 */
+		public static ConversionException withProcessResult(String message, ProcessResult processResult) {
+			String messageIncludingErrorOutput = message;
+			if (processResult.errorOutput != null) {
+				messageIncludingErrorOutput += " (command output: " + processResult.errorOutput + ")";
+			}
+			return new ConversionException(messageIncludingErrorOutput, processResult.exception);
+		}
```

Comment:
For the reviewer: I added this as I spent quite some time investigating what went wrong. I got an error, but no exception was thrown. Eventually, I noticed that the `ProcessResult#errorOutput` field contained something that actually help me to resolve the issue